### ISSUE TITLE
Use with React 0.14.X causes warnings

### ISSUE THREAD

**Issue by Jakehp** [kirjs/react-highcharts#8]
These were specifically found on React 0.14.0-alpha3
```
Warning: require("react/addons") is deprecated.
Access using require("react/addons/{addon}") instead.

Warning: DIV.getDOMNode(...) is deprecated. Please use React.findDOMNode(instance) instead.
```
This is more of a warning/heads up, than an actual issue. I'm sure you were planning on making the appropriate changes once 0.14 is out of alpha+beta.

**Comment by kirjs** [comment]
thanks Jake, I'll keep it open once 0.14 becomes a current release

**Comment by Kureev** [comment]
@kirjs `DIV.getDOMNode(...) is deprecated. Please use React.findDOMNode(instance) instead` could be fixed already today. `getDOMNode(...)` is deprecated since 0.13

**Comment by Kureev** [comment]
furthermore, in `0.14` facebook changed the way `refs` works: it'll return link to DOM element instead of element. [Read more](https://facebook.github.io/react/blog/2015/07/03/react-v0.14-beta-1.html#dom-node-refs)

**Comment by kirjs** [comment]
Should be resolved by #20
